src/main/resources/io/jenkins/plugins/file_parameters/Base64FileParameterDefinition/help.html

```diff
@@ -0,0 +1,41 @@
+<div>
+  <p>File parameter compatible with Pipeline, how to use it in a declarative pipeline:</p>
```

Comment:
The critical thing to mention here is that this parameter type is only suitable for small files.